src/index.js

```diff
@@ -1,29 +1,71 @@
+/* eslint-disable no-use-before-define */
 import './styles.css';
+import { addNewTask, form, input } from './addNewTask.js';
+import removeTask from './removeTask.js';
 
 const placeholder = document.querySelector('ul');
 
-const simpleTodoTasks = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'complete To Do list project',
-    completed: false,
-    index: 0,
-  },
-];
+const simpleTodoTasks = JSON.parse(localStorage.getItem('task')) || [];
 
 const populateEachTask = (arr) => {
+  localStorage.setItem('task', JSON.stringify(simpleTodoTasks));
+  placeholder.innerHTML = '';
   for (let i = 0; i < arr.length; i += 1) {
     const taskDetails = arr[i];
     const taskContainer = document.createElement('li');
+    taskContainer.setAttribute('data-id', i);
     taskContainer.className = 'task-container';
-    taskContainer.innerHTML = `<span class='task-content'><input type='checkbox'></span><span class='task-content'>${taskDetails.description}</span><span class='task-content index'><i class='fa fa-ellipsis-v'></i></span>`;
+    taskContainer.innerHTML = `<span class='task-content'><input type='checkbox' data-check></span><span class='task-content description'>${taskDetails.item}</span>
+    <input class='edit hide' value=${taskDetails.item}>
+    <span class='task-content index'></i></span>`;
+
+    const editBtn = document.createElement('i');
+    editBtn.className = 'fa';
+    editBtn.classList.add('fa-pencil');
+    taskContainer.appendChild(editBtn);
+
+    const removeBtn = document.createElement('i');
+    removeBtn.className = 'fa';
+    removeBtn.classList.add('fa-trash');
+    removeBtn.addEventListener('click', removeList);
+
+    taskContainer.appendChild(removeBtn);
     placeholder.appendChild(taskContainer);
+
+    editBtn.addEventListener('click', editDescription);
   }
-  return placeholder;
 };
 
-populateEachTask(simpleTodoTasks);
+form.addEventListener('submit', () => {
+  addNewTask(simpleTodoTasks, input.value);
+  populateEachTask(simpleTodoTasks);
+});
+
+const removeList = (e) => {
+  const li = e.target.closest('li');
+  const id = li.getAttribute('data-id');
+  removeTask(simpleTodoTasks, id);
+  populateEachTask(simpleTodoTasks);
+};
```

Comment:
- [ ] Nice job so far. There's a little requirement you're yet to implement. Whenever a todo is removed, the entire remaining todos indexes should get updated. They should be reordered in a way that, the indexes would be in order. This would prevents duplicate and unorder indexes. Kindly look into it and fix.

### Project Requirement
![project-rqt](https://user-images.githubusercontent.com/26597432/236165541-2988d192-7366-4fbf-baeb-54cbd93f036b.png)


### Your App Behaviour at the moment
[app-behave-2.webm](https://user-images.githubusercontent.com/26597432/236165634-a29f388d-87f1-47a9-a091-ade2f8bb83d6.webm)